sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java

```diff
@@ -61,21 +62,23 @@ TokenResult getCachedToken(String partitionName) {
         return getCachedToken(partitionName, false);
     }
 
-    TokenResult getCachedToken(String partitionName, Boolean includeExpiredToken) {
+    TokenResult getCachedToken(String partitionName, boolean includeExpiredToken) {
         TokenResult token = Utils.getGson().fromJson(SharedPreferencesManager.getString(partitionName), TokenResult.class);
-        if (token != null && !includeExpiredToken) {
-            Calendar utcCalendar = Calendar.getInstance(TimeZone.getTimeZone("GMT"));
+        if (token != null) {
+            if (!includeExpiredToken) {
 
-            /* The token is expired. */
-            if (utcCalendar.getTime().compareTo(token.expiresOn()) > 0) {
-                AppCenterLog.warn(LOG_TAG, String.format("Cached token result is expired for partition '%s'", partitionName));
-                return null;
+                /* If the token is expired. */
+                Calendar utcCalendar = Calendar.getInstance(TimeZone.getTimeZone("GMT"));

```

Comment:
Getting "GMT" timezone calendar and name it `utcCalendar`?

Reply:
Not my change and planning to get rid of Calendar here.

Reply:
well GMT is a timezone, this is the difference.
https://www.timeanddate.com/time/gmt-utc-time.html